### PR TITLE
Kubernetes provider 1.1.0 release notes

### PR DESCRIPTION
**Pull request by linglingye001**

### CODE REVIEW

**Review by RichardChen820 — CHANGES_REQUESTED** [review]

File: releaseNotes/KubernetesProvider.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.1.0 - xx xx, 2023
```
Comment (juniwang) [inline releaseNotes/KubernetesProvider.md:5]: is this date intentional?

File: releaseNotes/KubernetesProvider.md [hunk 2]
```diff
@@ -2,6 +2,19 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.1.0 - xx xx, 2023
+
+### Enhancements
+* Added the `auth.workloadIdentity.managedIdentityClientIdReference` property for workloadIdentity authentication. It enables the retrieval of the client ID of a user-assigned managed identity from a ConfigMap. [#812](https://github.com/Azure/AppConfiguration/issues/812)
```
Comment (juniwang) [inline releaseNotes/KubernetesProvider.md:8]: just to confirm, do we only support user-assigned managed identity? what about system-assigned or service principal?

File: releaseNotes/KubernetesProvider.md (hunk 1 above)
Comment (linglingye001) [reply releaseNotes/KubernetesProvider.md:5]: Yes, the final date will be updated before provider release.

File: releaseNotes/KubernetesProvider.md (hunk 2 above)
Comment (linglingye001) [reply releaseNotes/KubernetesProvider.md:8]: For now, we only support `ManagedIdentityClientIdReference` for workload identity, which is asked by the customer. If system-assigned managed identity is used for auth, users don's need to set `spec.auth` filed. And we have `servicePrincipalReference` for service principal. We could add configMapReference for `auth` later if there's more voices ask for it.

File: releaseNotes/KubernetesProvider.md [hunk 3]
```diff
@@ -2,6 +2,19 @@
...
+### Enhancements
+* Added the `auth.workloadIdentity.managedIdentityClientIdReference` property for workloadIdentity authentication. It enables the retrieval of the client ID of a user-assigned managed identity from a ConfigMap. [#812](https://github.com/Azure/AppConfiguration/issues/812)
+    ```
+    auth:
+      workloadIdentity:
+        managedIdentityClientIdReference:  
+          configMap: configmap-contains-managed-identity-client-id
+          key: key-of-managed-identity-client-id
+    ```
+* Added the `configMapData.separator` property for generating hierarchical format of json/yaml type ConfigMap data. It defaults to empty and should be configured only if the configuration file loader used in your application can't load key-values without converting them to the hierarchical format. [#834](https://github.com/Azure/AppConfiguration/issues/834)
```
Comment (zhenlan) [inline releaseNotes/KubernetesProvider.md:16]: Added support for outputting the ConfigMap data in hierarchical format when the type is set to `json` or `yaml` by specifying the new property `configMapData.separator`. This feature is useful if the configuration file loader used in your application can't load keys without converting them to the hierarchical format.

File: releaseNotes/KubernetesProvider.md [hunk 4]
```diff
@@ -2,6 +2,19 @@
...
+
+### Enhancements
+* Added the `auth.workloadIdentity.managedIdentityClientIdReference` property for workloadIdentity authentication. It enables the retrieval of the client ID of a user-assigned managed identity from a ConfigMap. [#812](https://github.com/Azure/AppConfiguration/issues/812)
+    ```
+    auth:
+      workloadIdentity:
+        managedIdentityClientIdReference:  
+          configMap: configmap-contains-managed-identity-client-id
+          key: key-of-managed-identity-client-id
+    ```
```
Comment (zhenlan) [inline releaseNotes/KubernetesProvider.md:15]: I think it's fine not to have this example. If you like you can include a link to the documentation for more information.

File: releaseNotes/KubernetesProvider.md [hunk 5]
```diff
@@ -2,6 +2,15 @@
...
 [Image][image] | [Sample][sample]
 
+## 1.1.0 - xx xx, 2023
+
+### Enhancements
+* Added the `auth.workloadIdentity.managedIdentityClientIdReference` property for workloadIdentity authentication. It enables the retrieval of the client ID of a user-assigned managed identity from a ConfigMap. [#812](https://github.com/Azure/AppConfiguration/issues/812)
+* Added support for outputting the ConfigMap data in hierarchical format by specifying the new property `configMapData.separator` if the ConfigMap is consumed as a mounted file. This feature is useful if the configuration file loader used in your application can't load keys without converting them to the hierarchical format. [#834](https://github.com/Azure/AppConfiguration/issues/834)
+
+### Bug fixes
+* Fixed `nil pointer deference` error when a key-value is pulled from App Configuration that has a `null` value. [#848](https://github.com/Azure/AppConfiguration/issues/848)
```
Comment (zhenlan) [inline releaseNotes/KubernetesProvider.md:12]: Hope this better described the symptom:

Fixed a bug that may cause the Kubernetes provider to crash with an `invalid memory address or nil pointer dereference` error when a key-value pulled from App Configuration has a `null` value.

**Review by zhenlan — APPROVED** [review]

**Review by RichardChen820 — APPROVED** [review]

**Pull request merged**